### ISSUE TITLE
ImportError: cannot import name 'pystone' help

### ISSUE THREAD

**Issue by 232824787** [ruiminshen/yolo2-pytorch#5]
Traceback (most recent call last):
  File "detect.py", line 27, in <module>
    import pybenchmark
  File "/home/a/anaconda3/lib/python3.6/site-packages/pybenchmark/__init__.py", line 1, in <module>
    from .profile import profile, stats, kpystones, pystones
  File "/home/a/anaconda3/lib/python3.6/site-packages/pybenchmark/profile.py", line 3, in <module>
    from test import pystone            # native python-core "PYSTONE" Benchmark Program
ImportError: cannot import name 'pystone'

**Comment by 232824787** [comment]
reinstall python is ok

**Comment by xmfbit** [comment]
@232824787 Hi, I have run into the same problem. Could you share your solution with more detail? The error of mine is as following:
```
Traceback (most recent call last):
  File "detect.py", line 33, in <module>
    import pybenchmark
  File "/home/me/anaconda2/envs/pyyolo/lib/python3.5/site-packages/pybenchmark/__init__.py", line 1, in <module>
    from .profile import profile, stats, kpystones, pystones
  File "/home/me/anaconda2/envs/pyyolo/lib/python3.5/site-packages/pybenchmark/profile.py", line 3, in <module>
    from test import pystone            # native python-core "PYSTONE" Benchmark Program
ImportError: cannot import name 'pystone'
```

**Comment by rosenfeldamir** [comment]
I ran into the same problem....

**Comment by diggerdu** [comment]
Quick Fix: You can checkout a copy of pystone.py from the cpython github repository and copy it to a test directory that is present in your PYTHONPATH. Or you would copy it to a test directory in your python project's root (ugly way).

Perhaps Anaconda Python does not ship with a copy of the test module. This is a standard part of Python 2.7. Other times, users accidentally overwrite their Python standard library's test module with something else. You can try to use the Python version that is shipped with OS X instead. If that fails as well, then try to see which test module is being loaded, and go from there.

**Comment by xmfbit** [comment]
@diggerdu Thanks for your kindness. I have solved the problem and forgot to write my solution down. My solution was just the same with your second solution and I indeed used Anaconda.

**Comment by jacktkk** [comment]
@xmfbit 
hi~ sorry for bothering u
the pystone.py I found only match python2,
so can u tell me where I can find the proper file(pystone.py)?
best wishes!

**Comment by diggerdu** [comment]
@jacktkk 
I post a [gist](https://gist.github.com/diggerdu/27a46a8d130a222999f160288940c61c)

**Comment by jacktkk** [comment]
@diggerdu thx~

**Comment by jacktkk** [comment]
@diggerdu do you know how to fix the problem #8 ?
